### DISCUSSION TITLE
Range component look reversed

### DISCUSSION THREAD

**Discussion by tmaabm** [saadeghi/daisyui#2470] [General]
When I use the _range_ component, the slider looks are reversed compared to how they look on the daisyUI website

| daisyUI    | Locally |
| -------- | ------- |
| ![image](https://github.com/saadeghi/daisyui/assets/16197728/53a3d9ce-a49c-4a29-a830-60c18788cd34) |![image](https://github.com/saadeghi/daisyui/assets/16197728/9a3c24df-7577-486a-b4e4-a81a7b023ff4)   |

As you can see, the transparent section is reversed. The JSX I'm using is identical to the example given on the daisyUI documentation.

**Comment by saadeghi** [comment]
Please share a code example. Otherwise it's hard to guess what is causing the issue.

**Reply by tmaabm** [reply]
The exact code is:

```js
      <Input
          label={name}
          type="range"
          min="0"
          max="100"
          class="range"
          step="1"
          {...register(name)}
      />
```
where `<Input />` is defined as:

```js
const Input = forwardRef(
  ({ t, error, label, className = "", ...props }, ref) => {
    return (
      <label class="input-group input-group-vertical">
        <span>{label}</span>
        <input
          type="text"
          placeholder={label}
          class={clsx(`input input-bordered w-full ${className}`, {
            "input-error": isDefined(error),
          })}
          aria-invalid={error ? "true" : "false"}
          ref={ref}
        />
        <ErrorMessage error={error?.message} t={t} />
      </label>
    );
  },
);
```
---
**N.B** ` {...register(name)}` is from `react-hook-form`

**Reply by saadeghi** [reply]
The code seems okay: https://play.tailwindcss.com/2a3IcE27Zq

Are you sure there's no other CSS styles affecting your page?

**Reply by tmaabm** [reply]
Thanks; I'll take a deeper look and report back

**Reply by tmaabm** [reply]
The issue was caused by the use of the option `rtl: boolean` and [tailwindcss-flip](https://github.com/cvrajeesh/tailwindcss-flip). Once I removed _both_, the styles were fixed.
